x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -69,7 +71,7 @@ export const TopValues: FC<Props> = ({
   } = useDataVisualizerKibana();
 
   if (stats === undefined || !stats.topValues) return null;
-  const { fieldName, sampleCount } = stats;
+  const { fieldName, sampleCount, approximate } = stats;
```

Comment:
Did you decide not to show an 'Other' row for cases like this?

<img width="681" alt="Screenshot 2024-05-14 at 11 28 58" src="https://github.com/elastic/kibana/assets/7405507/6729d276-2d17-4ea3-80e5-07c47ca0e451">

Reply:
Yes for now `Other` would be hard to calculate for ES|QL currently doesn't support tracking total hits. And we cannot base it off the cardinality/count because the requests are randomly sampled, and therefore can change.